docs/superpowers/specs/2026-08-16-auto-offline-save-design.md

```
@@ -0,0 +1,217 @@
+### 案 B（採用）: `savedLaws` の keyPath を `[lawId, revisionId]` に変え、現行版フラグを持たせる
+
+単一ストアで版と現行版スロットを表現でき、一覧・エビクション・エクスポートの実装が 1 箇所に閉じる。
+keyPath 変更のため object store の作り直しが必要になるが、versionchange トランザクション内で旧レコードを読み出して詰め直せるため、情報の欠落なく移行できる。
+
+### 案 C: 過去版は本文ノードだけ残す
+
+メタ情報を持たないため過去版を一覧・削除できず、エビクションの対象にもできない。孤児ノードが溜まるだけなので却下。
+
+## データモデル（DB v4）
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)